### ISSUE TITLE
Lambda settings?

### ISSUE THREAD

**Issue by Heartsie** [yusuketomoto/chainer-fast-neuralstyle#37]
Hey guys does anyone have examples of what lambda_feat, lambda_style, and lambda_tv do? It would be great to actually see what changing these settings does to the model. Maybe one of them helps with the noise?

**Comment by cryptexis** [comment]
yes, they are measuring the strength of the style, content and spatial smoothness in the outcome image accordingly.

**Comment by Heartsie** [comment]
Ok interesting, are the default values good? I am not even sure what kind of range to use when testing these parameters out.

**Comment by cryptexis** [comment]
The paper only says about lambda_tv between 1e-4 and 1e-6....about others I don't know anything
